middleware/2fa.js

```diff
@@ -18,7 +18,7 @@ const changeMethods = ['confirm'];
 const DETERM_KEY_SEED = process.env.DETERM_KEY_SEED || creatorKeyJson.private_key;
 const MULTISIG_CONTRACT_HASHES = process.env.MULTISIG_CONTRACT_HASHES ? process.env.MULTISIG_CONTRACT_HASHES.split() :['7GQStUCd8bmCK43bzD8PRh7sD2uyyeMJU5h8Rj3kXXJk','AEE3vt6S3pS2s7K6HXnZc46VyMyJcjygSMsaafFh67DF', '45gayUD7tUKFc3vvGwzoPEeFS6RjdQWu7SHGpxAJe13F'];
 const CODE_EXPIRY = 300000;
-
+const oneHunTgas = '100000000000000'
```

Comment:
```
const NEAR_2FA_GAS = process.env. NEAR_2FA_GAS ||  '100000000000000' 
```

let's:
1) name based on purpose, not current value
2) use ALL_CAPS as customary for other constants.
3) allow to override in environment variable

Also let's not mix this into PR called "deprecate sendRecoveryMessage", as it's off-topic change.

Reply:
Got it, wasn't aware I was still on branch. Reverted!